pkg/core/native/native_neo.go

```diff
@@ -195,7 +212,13 @@ func (n *NEO) Initialize(ic *interop.Context) error {
 	if err != nil {
 		return err
 	}
-
+	price := big.NewInt(DefaultRegisterPrice)
+	si := state.StorageItem(bigint.ToBytes(price))
```

Comment:
We have `setIntWithKey` helper for this in `util.go`.

Reply:
Fixed.